sources/install/package_ad.sh

```diff
@@ -31,9 +31,16 @@ function install_responder() {
     colorecho "Installing Responder"
     git -C /opt/tools/ clone --depth 1 https://github.com/lgandx/Responder
     cd /opt/tools/Responder
+    cp -v /root/sources/assets/patches/responder.patch responder.patch
+    git apply --verbose responder.patch
```

Comment:
https://stackoverflow.com/questions/70870041/cannot-import-name-mutablemapping-from-collections

Reply:
Do you think it would be relevant to propose this in a pull request to the official repo? It would allow Responder to be compatible with python3.10 if I understood the issue correctly.
It would also allow to hot merge during the build instead of keeping the static .patch file. What do you think?